src/AccessibilityInsights.SharedUx/Controls/TextRangeControl.xaml

```diff
@@ -64,6 +64,12 @@
                                         <DataTrigger Binding="{Binding Path=vmHilighter.State, Mode=OneWay, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType=local:TextRangeControl}}" Value="Off">
                                             <Setter Property="GlyphName" Value="UnSetColor"/>
                                         </DataTrigger>
+                                        <DataTrigger Binding="{Binding RelativeSource={RelativeSource AncestorType=Button,AncestorLevel=1}, Path=IsMouseOver}" Value="True">
```

Comment:
I would have preferred to use the `hoverAwareFabricIconOnUnthemedButtonParent` style here, but the multiple triggers prevented me from doing so. Maybe you know of a clever way to avoid the duplication?

Reply:
Unfortunately, I am not aware of a native way to merge two styles' triggers in XAML :/